### PR TITLE
[MOO-523] Add docs for dangling references (MX8)

### PR DESCRIPTION
**Pull request by govorov**

### CODE REVIEW

**Review by ilya-kurganskii — APPROVED** [review]
LGTM 👍

**Comment by Adam-Dupaski** [comment]
@ConnorLand please verify if this corresponds to a future release

**Comment by ConnorLand** [comment]
> @ConnorLand please verify if this corresponds to a future release

Will do.

**Comment by ConnorLand** [comment]
> @ConnorLand please verify if this corresponds to a future release

It will be for a subsequent 8.18.x release, probably the next one. I have added a label accordingly.

File: content/refguide8/offline-first.md [hunk 1]
```diff
@@ -130,7 +124,7 @@ Synchronization requires a connection to the server, so during synchronization,
 
 The synchronization is atomic, which means that either everything or nothing is synchronized. Exceptions are described in the [Model- or Data-Related Errors](#othererrors) section below.
 
-If a network error happens during the file upload (via [step 2 in the upload phase](#steptwo)), Mendix retries to upload the failed files. If there is an error for the second time, the synchronization is aborted. The changes at that moment are kept on the local device, so it can be retried later.
+If a network error happens during the file upload (via [step 2 in the upload phase](#steptwo)), Mendix tries to upload the failed files again. If there is an error for the second time, the synchronization is aborted. The changes at that moment are kept on the local device, so it can be retried later.
```
Comment (ConnorLand) [inline content/refguide8/offline-first.md:127]: @govorov I see that several changes have been made to 2.2.1 Upload phase section. The new text is fine. However, there used to be anchor links (for example: <a name="steptwo"></a>) that your commit had deleted. The line I'm commenting on still refers to that old, deleted anchor link ("via [step 2...](#steptwo))". Please address this by taking one of the actions, whichever you think is appropriate:
1. Re-implement the anchor links in a way that fits with your new content.
2. Delete references to anchor links (such as "#steptwo" above) while making sure those sentences still make sense.

Comment (govorov) [reply content/refguide8/offline-first.md:127]: Fixed the anchors, thanks for pointing out 👍

Comment (ConnorLand) [reply content/refguide8/offline-first.md:127]: Thanks so much Stas.

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**